libclc/generic/include/clcmacro.h

```diff
@@ -1,3 +1,6 @@
+#ifndef __CLC_MACRO_H
```

Comment:
I assume this file should also has copyright header.
```c++
//===----------------------------------------------------------------------===//
//
// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
// See https://llvm.org/LICENSE.txt for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
//
//===----------------------------------------------------------------------===//
```

Could you apply this change to new files as well, please? E.g. libclc/generic/include/math/half_unary.inc.

Reply:
yes, I probably should. Note there is no new file in this patch, I forgot to delete the one in the lib side for `half_unary.inc`

Reply:
I updated the files, I also removed `half_unary.inc` it was a left over of the refactoring